v-next/hardhat-network-helpers/src/internal/network-helpers/time/time.ts

```diff
@@ -27,10 +24,12 @@ export class Time implements TimeI {
   }
 
   public async increase(amountInSeconds: NumberLike): Promise<number> {
+    await this.#networkHelpers.throwIfNotDevelopmentNetwork();
```

Comment:
Is the reason why we have to call it explicitly because interaction with the `NetworkHelpers` inside the `increase` implementation happen too late and we want to get the error as early as possible?

Reply:
Not sure I fully understand the question, but I believe that the answer I gave in a previous comment also works here. The method `throwIfNotDevelopmentNetwork` must be called when a method is used, and not during the class initialization

Reply:
Here, I was thinking what if we didn't `throwIfNotDevelopmentNetwork` at all and just waited for it to fail when we call something on the `networkHelpers`. But I'm assuming we want to prevent this call - https://github.com/NomicFoundation/hardhat/blob/48d64ebed36dac283593e18e75db3348705cc630/v-next/hardhat-network-helpers/src/internal/network-helpers/time/helpers/increase.ts#L15 - from happening in the first place. Which answers my question.

Reply:
Correct, we want to prevent that call from happening